src/foundry/client/config.d.ts

```diff
@@ -1893,6 +1895,11 @@ interface CanvasGroupConstructor extends PixiContainerConstructor {
 
   /**
    * The name of this canvas group
+   * @remarks Not used in EffectsCanvasGroup in v11
    */
-  groupName: string;
+  groupName?: string;
 }
+
+// TODO: Replace after https://github.com/microsoft/TypeScript/issues/17867 is resolved
```

Comment:
Wrong issue. I don't think there's a related issue to this, the problem is largely on Foundry's end;
```ts
class A {
    constructor(foo: string);
}

class B extends A {
    constructor(foo: string, bar: number);
}
```

Class `B` isn't properly assignable to class `A` because of the extra constructor parameter. Same thing here with the `sprite` property.

They just don't use it this way.